templates/zerver/for/working-groups-and-communities.md

```diff
@@ -0,0 +1,37 @@
+Working groups and communities need different things out of chat than
+companies do. At a company, users are on their company's chat all day. In a
+part-time working group or community, users might check the chat less than
```

Comment:
In particular, I think for a lot of these trade groups, a lot of high-value users are checking Zulip 1-3x/week, not 1-3x / month; the key thing is that they're able to do something useful with 10 minutes when they were offline for the last 2 days and 300 messages were sent in the meantime.